rest_api/v2/info/index.yml

```
@@ -3,21 +3,55 @@ version: "v2"
-  # Getting started
-  Here is the documentation on Partoo Rest API. It is meant for querying and updating data in Partoo database using `JSON` format.
+  # Introduction
+  The Partoo Rest API allows you to automate all the actions that are possible to do in the Partoo Web Application.
 
-  ## Purposes
-  Partoo Rest API can be used for many differents purposes:
-    - Retrieve data on businesses if you are a **publisher**
-    - Manage your businesses/users if you are a **client**
-    - Create/Subscribe/Manage organisations, users & businesses if you are a **reseller**
+  The Partoo Rest API can be used for many different purposes:
+    - Create/update/delete your businesses & users if you are a client.
+    - Create/subscribe/manage organisations, businesses & users if you are a reseller.
+    - Retrieve data on businesses you have access to if you are a publisher.
     - ...
 
   # JS SDK
+  To know how to integrate Partoo in a white label mode using our JS SDK, please check our [JS SDK documentation](/js_sdk/).
+
+  # Getting started
+
+  ## Security Disclaimer
+  **The Partoo Rest API is meant to be used only from a server that you control.** The API Key should **never** be transmitted
+  to the front-end, and should be kept secret.
+  If you require to make calls from a front-end, you should proxy them by a server that you control, that will add the API Key.
+
+  ## Environments
+  We provide two public environments, **Production** and **Sandbox**.
+  - **Production** is where your final data should be.
+  - **Sandbox** is a playground environment for you to test your API integration without having to worry about side effects.
+
+  Some functionalities are disabled in Sandbox, to avoid having a production impact:
+  - Business data will not be sent to the different publishers as GMB, Facebook, etc.
+  - It's not possible to import GMB and Facebook account listings.
+  - There's no real reviews data, or real analytics data. The account starts empty, but can be filled by fake data if needed. For this, contact you Partoo Account Manager.
+
+  ## First steps
+  First, it's a good idea to start building and testing your integration in the Sandbox environment.
 
-  To know how to integrate Partoo views using our JS SDK, please check our [JS SDK documentation](/js_sdk/).
+  Ask your Partoo Account Manager to send you an API Key for Sandbox. You'll need to pass this API Key in all your API calls in order
+  to be authentified, in the header `x-APIKey`. More details in the <a href="#section/Authentication">Authentication section</a>.
 
-  # Resources structure
+  From there, you can call any route that your user has access to. To know the exact API endpoint to use, you can click
+  on the route url above the "Response samples", and this will give you the urls for Production and Sandbox.
+
+  If you require an UI access to Sandbox, ask your Partoo Account Manager to give you one. Beware that Sandbox environment will be empty by default,
```

```suggestion
  If you require a UI access to Sandbox, ask your Partoo Account Manager to give you one. Beware that Sandbox environment will be empty by default,
```
https://www.a-or-an.com/a_an/ui